hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-soak.yaml

```diff
@@ -145,6 +145,7 @@
             post-env: ''
             soak-deploy: ''
             soak-continuous: ''
+            cron-string: 'H */6 * * *'
```

Comment:
is there harm in continuing to run 1.1 soak frequently? the cluster is already up, so we're not saving on resource costs.